tests/components/ozw/test_services.py

```diff
@@ -66,8 +66,32 @@ async def test_services(hass, lock_data, sent_messages, lock_msg, caplog):
     await hass.services.async_call(
         "ozw",
         "set_config_parameter",
-        {"node_id": 10, "parameter": 6, "value": 2147483657},
+        {"node_id": 39, "parameter": 3, "value": 2147483657},
         blocking=True,
     )
     assert len(sent_messages) == 3
-    assert "Value 12 out of range for parameter 1" in caplog.text
+    assert "Value 2147483657 out of range for parameter 3" in caplog.text
+
+    # Test set_config_parameter short
+    await hass.services.async_call(
+        "ozw",
+        "set_config_parameter",
+        {"node_id": 39, "parameter": 81, "value": 3000},
+        blocking=True,
+    )
+    assert len(sent_messages) == 4
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5985897)